EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/Router/MessageRouter.cs

```diff
@@ -0,0 +1,69 @@
+using Discord.WebSocket;
+using HK47.MessageHandlers.Interfaces;
+using HK47.Services;
+
+namespace HK47.Router;
+
+public class MessageRouter(IDataService dataService, ISteamMessageHandler steamHandler, IGeneralMessageHandler generalHandler, IMessageService messageService) : IMessageRouter
+{
+    public async Task ProcessMessage(SocketMessage message)
+    {
+        try
+        {
+            Console.WriteLine("Processing Message...");
+            if (!message.Content.StartsWith('.'))
+            {
+                // Ignore message.
+                Console.WriteLine("Ignoring Message....");
```

Comment:
Another thing we could consider (not part of this PR - just adding a note so I can make an issue) would be creating a resource file for strings (more so where the bot has text not debug console output), that way we can reuse text as well as keep in sync when we get around to unit testing.